### PR TITLE
add photo block

### PR DESCRIPTION
**Pull request by iHateAntichrist**
https://prnt.sc/p1dXNijOfmTk

### CODE REVIEW

File: src/components/PhotosBlock/PhotosBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+  // const loadPhones = async () => {
+  //   const data = await getPhones();
+
+  //   setPhones(data);
+  // };
+
+  // useEffect(() => {
+  //   loadPhones();
+  // }, []);
```
Comment (Stambuliak) [inline src/components/PhotosBlock/PhotosBlock.tsx:43]: do we need this?

File: src/components/PhotosBlock/PhotosBlock.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
+.photo-block {
+  position: relative;
+  width: 559px;
+  height: 471px;
```
Comment (Anna-Kuzmenko) [inline src/components/PhotosBlock/PhotosBlock.scss:4]: You don't need this styles, it should be grid

File: src/components/PhotosBlock/PhotosBlock.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    }
+  }
+
+  &__selectedItem {
+    position: absolute;
+    top: 11px;
+    right: 11px;
+
+    width: 442px;
+    height: 442px;
```
Comment (Anna-Kuzmenko) [inline src/components/PhotosBlock/PhotosBlock.scss:32]: these styles only for desktop, we should start from phone

**Review by Anna-Kuzmenko — CHANGES_REQUESTED** [review]
Styles only for desktop version, you should start from phone version

**Review by VenomMB — CHANGES_REQUESTED** [review]

**Review by VenomMB — APPROVED** [review]

**Pull request merged**